### PR TITLE
Add Norwegian landing page

### PR DESCRIPTION
**Pull request by vonElfvin**

### CODE REVIEW

File: src/pages/Embark/Landing.tsx [hunk 1]
```diff
@@ -147,26 +149,61 @@ export const Landing: React.FC<{ language?: string }> = ({ language }) => {
...
+                  <Paragraph>{textKeys.STARTPAGE_INSURED_BODY()}</Paragraph>
+                  <ProceedButton
+                    size="lg"
+                    to={(language ? '/' + language : '') + '/new-member/switch'}
+                  >
+                    {textKeys.STARTPAGE_INSURED_BUTTON()}
+                  </ProceedButton>
+                </Card>
+              </>
+            )}
```
Comment (palmenhq) [inline src/pages/Embark/Landing.tsx]: What about moving this stuff to separate components, like a `LandingContentSe` or so? Just to slim this one down a bit and separate the responsibilities.

**Review by palmenhq — COMMENTED** [review]
Awesome! Just one question really. And welcome to the web team! You may join @HedvigInsurance/www also, if you wanna ;)

File: src/pages/Embark/Landing.tsx (hunk 1 above)
Comment (vonElfvin) [reply src/pages/Embark/Landing.tsx]: I tried that but sending "textKeys" was a nightmare, I have no idea what type it is and don't want to implicitly cast it to any. I'll try it again though.

Comment (palmenhq) [reply src/pages/Embark/Landing.tsx]: I think you could put `const textKeys = useTextKeys()` inside these components also :)

File: src/pages/Embark/Landing.tsx [hunk 2]
```diff
@@ -150,63 +150,74 @@ export const Landing: React.FC<{ language?: string }> = ({ language }) => {
...
             )}
           </Container>
         </Wrapper>
       </LandingPageContainer>
     </Page>
   )
 }
+
+const LandingPageCardsSe: React.FC<{
+  textKeys: Record<string, any>
```
Comment (palmenhq) [inline src/pages/Embark/Landing.tsx]: I think we should use `TextKeyMap` from https://github.com/HedvigInsurance/web-onboarding/blob/master/src/utils/hooks/useTextKeys.ts - or just use `useTextKeys` inside these components instead.

**Review by palmenhq — APPROVED** [review]
💥🔥! Lets 🛳 it!

**Pull request merged**